packages/fxa-settings/src/index.tsx

```diff
@@ -6,7 +6,8 @@ import React from 'react';
 import { render } from 'react-dom';
 import AppErrorBoundary from 'fxa-react/components/AppErrorBoundary';
 import sentryMetrics from 'fxa-shared/lib/sentry';
-import App from './components/App';
+import GeneralizedApp from './components/GeneralizedRootApp';
```

Comment:
Nitpick, can we name this the same thing, either calling this `GeneralizedAppRoot` on import or renaming the component to `GeneralizedApp`? Or, just name this `App`, and rename the existing `App` to `SettingsRoot` or `AppSettings` or something like that, since we'll likely want to rename this later if we call it `GeneralizedRootApp`. Alternatively If we want to move all Settings components into a `/settings/` directory anyway, we could have `src/components/App/index.tsx` as well as one for `src/components/settings/App/index.tsx`.